query/physicalplan/aggregate.go

```diff
@@ -26,7 +26,8 @@ func Aggregate(
 	s *parquet.Schema,
 	agg *logicalplan.Aggregation,
 	final bool,
-) (*HashAggregate, error) {
+	ordered bool,
```

Comment:
Eventually, it might make sense to pass something similar to the execOpts or an execution context here.